workflows/dynamic_index.md

```diff
@@ -22,6 +22,15 @@ The program then simultaneously inserts newer points drawn from the file and del
 in chunks of `consolidate_interval` points so that the number of active points in the index is approximately `active_window`.
 It terminates when the end of data file is reached, and the final index has `active_window + consolidate_interval` number of points.
 
+The index also supports filters on steaming index, you can use `insert_point` function overloads to either insert points as before or insert points with labels.
+Additional options are added to support this in `apps/test_streaming_scenario` and `apps/test_streaming_scenario` please refer to program arguments for more details.
+
+---
+> Note
+* The index does not support mixed points, that is, either all points do not have filters or all points have filters. 
```

Comment:
you mean all points have labels or no labels?

Reply:
Yes. I changed the wording here as well.